ArduCopter/mode.cpp

```diff
@@ -508,14 +508,20 @@ void Mode::make_safe_spool_down()
  */
 int32_t Mode::get_alt_above_ground_cm(void)
 {
-    int32_t alt_above_ground;
-    if (!copter.get_rangefinder_height_interpolated_cm(alt_above_ground)) {
-        bool navigating = pos_control->is_active_xy();
-        if (!navigating || !copter.current_loc.get_alt_cm(Location::AltFrame::ABOVE_TERRAIN, alt_above_ground)) {
-            alt_above_ground = copter.current_loc.alt;
-        }
+    int32_t alt_above_ground_cm;
+    if (copter.get_rangefinder_height_interpolated_cm(alt_above_ground_cm)) {
+        return alt_above_ground_cm;
+    }
+    // .... 
```

Comment:
missing comment maybe?

Reply:
Yes - I'm hoping someone can fill it in ;-)

Reply:
maybe, "use alt-above-home if not navigation horizontally"

Reply:
I've just removed the elipsis.